subsys/bluetooth/controller/ll_sw/ll_filter.c

```diff
@@ -567,6 +567,11 @@ static void rpa_adv_refresh(void)
 		return;
 	}
 
+	idx = ll_rl_find(ll_adv->id_addr_type, ll_adv->id_addr, NULL);
+	if (idx == FILTER_IDX_NONE) {
```

Comment:
Can you use the same check we use elsewhere? `if(idx >= ARRAY_SIZE())`...

Reply:
Yes. will do.